EIPS/eip-3448.md

```diff
@@ -52,19 +47,17 @@ The goals of this effort have been the following:
 There are no backwards compatibility issues.
 
 ## Test Cases
```

Comment:
The actual test cases should be listed here or linked to.

Reply:
I added a solidity contract w/ the test cases